data-visualization/data-visualization.md

```diff
@@ -6,12 +6,12 @@
 
 ### Prerequisites
 
-- HTML
+- [HTML](/html/html.md)
 - CC
```

Comment:
@yati1998 , can you please tell me what CC stands for? :)

Reply:
It was already mentioned here. So Even I don't have any idea about it. I could remove it if you want so?

Reply:
Can you change it to a link for the CSS lesson?

Reply:
ohk

Reply:
nice @yati1998